### PR TITLE
Update publish-docker-package.yaml

### PR DESCRIPTION
**Pull request by markgoertz**

### CODE REVIEW

**Comment by markgoertz** [comment]
## Metrics: workflow

| Path                 | eval.auc   | eval.binary_accuracy   | eval.f1\_score   | eval.loss   | eval.precision   | eval.recall   | step   | train.auc   | train.binary_accuracy   | train.f1\_score   | train.learning_rate   | train.loss   | train.precision   | train.recall   |
|----------------------|------------|------------------------|-----------------|-------------|------------------|---------------|--------|-------------|-------------------------|------------------|-----------------------|--------------|-------------------|----------------|
| dvclive/metrics.json | 0.99409    | 0.9767                 | 0.97672         | 0.11913     | 0.97595          | 0.97749       | 29     | 0.99965     | 0.9963                  | 0.9963           | 0.0001                | 0.0485       | 0.99485           | 0.99776        |

| Path                | Param                     | HEAD   | workspace                     |
|---------------------|---------------------------|--------|-------------------------------|
| dvclive/params.yaml | model.activation          | -      | relu                          |
| dvclive/params.yaml | model.batch_size          | -      | 32                            |
| dvclive/params.yaml | model.epochs              | -      | 30                            |
| dvclive/params.yaml | model.input_features      | -      | 1                             |
| dvclive/params.yaml | model.input_shapes.ACC    | -      | 256                           |
| dvclive/params.yaml | model.input_shapes.BVP    | -      | 256                           |
| dvclive/params.yaml | model.input_shapes.EDA    | -      | 32                            |
| dvclive/params.yaml | model.input_shapes.TEMP   | -      | 32                            |
| dvclive/params.yaml | model.kernel_regularizer  | -      | 0.001                         |
| dvclive/params.yaml | model.kernel_size         | -      | 3                             |
| dvclive/params.yaml | model.learning_rate       | -      | 0.001                         |
| dvclive/params.yaml | model.metrics             | -      | \['EDA', 'BVP', 'TEMP', 'ACC'] |
| dvclive/params.yaml | model.shuffle_buffer_size | -      | 1024                          |

## Plots

### Dataset distribution

**Comment by markgoertz** [comment]
## Metrics: workflow

| Path                 | eval.auc   | eval.binary_accuracy   | eval.f1\_score   | eval.loss   | eval.precision   | eval.recall   | step   | train.auc   | train.binary_accuracy   | train.f1\_score   | train.learning_rate   | train.loss   | train.precision   | train.recall   |
|----------------------|------------|------------------------|-----------------|-------------|------------------|---------------|--------|-------------|-------------------------|------------------|-----------------------|--------------|-------------------|----------------|
| dvclive/metrics.json | 0.99334    | 0.97867                | 0.97861         | 0.1212      | 0.98172          | 0.97551       | 29     | 0.9995      | 0.99509                 | 0.9951           | 0.0001                | 0.05429      | 0.99331           | 0.9969         |

| Path                | Param                     | HEAD   | workspace                     |
|---------------------|---------------------------|--------|-------------------------------|
| dvclive/params.yaml | model.activation          | -      | relu                          |
| dvclive/params.yaml | model.batch_size          | -      | 32                            |
| dvclive/params.yaml | model.epochs              | -      | 30                            |
| dvclive/params.yaml | model.input_features      | -      | 1                             |
| dvclive/params.yaml | model.input_shapes.ACC    | -      | 256                           |
| dvclive/params.yaml | model.input_shapes.BVP    | -      | 256                           |
| dvclive/params.yaml | model.input_shapes.EDA    | -      | 32                            |
| dvclive/params.yaml | model.input_shapes.TEMP   | -      | 32                            |
| dvclive/params.yaml | model.kernel_regularizer  | -      | 0.001                         |
| dvclive/params.yaml | model.kernel_size         | -      | 3                             |
| dvclive/params.yaml | model.learning_rate       | -      | 0.001                         |
| dvclive/params.yaml | model.metrics             | -      | \['EDA', 'BVP', 'TEMP', 'ACC'] |
| dvclive/params.yaml | model.shuffle_buffer_size | -      | 1024                          |

## Plots

### Dataset distribution
